### PR TITLE
Ark docs

### PR DESCRIPTION
**Pull request by patrikpordi**
Added/ Updated ARK Hardware in the Docs
Tested it, looks nice on my end.

Let me know if anything has to change.
Thanks in advance for the merge!

### CODE REVIEW

**Comment by patrikpordi** [comment]
Added @dakejahl

**Review by dakejahl — APPROVED** [review]

File: en/dronecan/ark_cannode.md [hunk 1]
```diff
@@ -12,6 +12,11 @@ Order this module from:
 
 - [ARK Electronics](https://arkelectron.com/product/ark-cannode/) (US)
 
+Find more information:
+
+- [ARK CANnode Documentation](https://arkelectron.gitbook.io/ark-documentation/sensors/ark-cannode)
```
Comment (hamishwillee) [inline en/dronecan/ark_cannode.md:17]: I'm going to move these to "See also" sections.

File: en/gps_compass/rtk_gps.md [hunk 2]
```diff
@@ -22,7 +22,8 @@ It also highlights devices that connect via the CAN bus, and those which support
 
 Device | GPS | Compass | [DroneCAN](../dronecan/index.md) | [GPS Yaw](#configuring-gps-as-yaw-heading-source) | PPK
 :--- | :---: | :---:  | :---:  | :---:  | :---:
-[ARK RTK GPS](https://arkelectron.com/product/ark-rtk-gps/) | F9P  | ICM42688p | &check; | [Dual F9P][DualF9P] |
+[ARK RTK GPS](../dronecan/ark_rtk_gps.md) | F9P  | BMM150 | &check; | [Dual F9P][DualF9P] |
+[ARK MOSAIC-X5 RTK GPS](../dronecan/ark_mosaic__rtk_gps.md) | Mosaic-X5  | IIS2MDC | &check; | [Septentrio Dual Antenna][SeptDualAnt] |
```
Comment (hamishwillee) [inline en/gps_compass/rtk_gps.md:26]: @patrikpordi Is this really Septentrio Dual Antenna ? If so, perhaps link to that information from the ARK MOSAIC-X5 page.

Comment (hamishwillee) [reply en/gps_compass/rtk_gps.md:26]: Ignore that. First thing you said!!!

**Review by hamishwillee — APPROVED** [review]
Really nice job @patrikpordi .

I subedited for style and conventions but otherwise it is largely unchanged.

**Pull request merged**